packages/enum/README.md

```diff
@@ -2,13 +2,19 @@
 
 Small tagged-union library for TypeScript.
 
-[//]: # 'TODO'
-[//]: # '[Documentation](https://soramitsu.github.io/scale-codec-js-library/guide/enum)'
-
 ## Features
 
 - **Type narrowing** and **exhaustiveness check** (due to the library following TypeScript's [discriminated union pattern](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes-func.html#discriminated-unions))
-- Type-safe variants creation with type inference
+- Type-safe variants creation
+- Type inference on variants creation
```

Comment:
```suggestion
- Creation of type-safe variants
- Creation of type inference on variants
```

Reply:
First one is good, but the second is not valid. It means that the method to create a variant is able to infer its typing from the context. How to say it better?

Reply:
would this work?

```suggestion
- Creation of type-safe variants
- Type inference from variant creation
```